modules/nf-core/bcftools/pluginsplit/main.nf

```diff
@@ -15,17 +15,18 @@ process BCFTOOLS_PLUGINSPLIT {
     path(targets)
 
     output:
-    tuple val(meta), path("*.{vcf,vcf.gz,bcf,bcf.gz}"), emit: vcf
-    tuple val(meta), path("*.tbi")                    , emit: tbi, optional: true
-    tuple val(meta), path("*.csi")                    , emit: csi, optional: true
-    path "versions.yml"                               , emit: versions
+    tuple val(meta), path("*/*.{vcf,vcf.gz,bcf,bcf.gz}"), emit: vcf
```

Comment:
Could you stage the input file in a folder instead to fix the first issue? 

E.g.: `tuple val(meta), path(files, stageAs: "inputs/*")`